LICENSE.BINARY

```diff
@@ -794,176 +794,170 @@ BINARY/EXTENSIONS/druid-kafka-eight
 BINARY/WEB-CONSOLE
 The following dependency names are NPM package names (https://www.npmjs.com).
 
-    This product bundles @babel/runtime version 7.3.4, copyright Sebastian McKenzie and other contributors,
+    This product bundles @babel/runtime version 7.4.3, copyright Sebastian McKenzie,
```

Comment:
Where's 7.4.3 from? I see it in `web-console/package-lock.json` as version 7.4.5.

Reply:
Dang, 7.4.3 is used in 0.15.0. Didn't notice this version was bumped up again. 
Looks like this PR needs to be adjusted to master. I will close this and raise another PR for 0.15.0.